CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ When `TYPE` can be:
 - **fix** - a bug fix
 - **refactor** - code change that neither fixes a bug nor adds a feature
 
-**All PRs must include a commit message with the changes description!**
+**All PRs must include a commit message with the a description of the changes!**
```

Comment:
this is incorrect grammar

Reply:
"All PRs must include a commit message with the description of the changes made" would be accurate